terraform/gitops/generate-files/templates/vnext/kustomization.yaml.tpl

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - istio-config.yaml
+  - switch-jws-deployment.yaml
+  - vault-secret.yaml
+helmCharts:
+- name: vnext
+  releaseName: ${vnext_release_name}
+  version: ${vnext_chart_version}
+  repo: ${vnext_chart_repo}
+  valuesFile: values-vnext.yaml
+  namespace: ${vnext_namespace}
+- name: elasticsearch
```

Comment:
Isn't it a stateful resource?

Reply:
yes, but we haven't added support for it in the stateful resources module, since we have dropped elastic search in favor of the grafana stack.